src/main/scala/rocks/muki/graphql/codegen/CodeGenStyles.scala

```diff
@@ -36,7 +36,23 @@ object CodeGenStyles {
       graphQLQueryFile,
       GraphQLQueryGenerator.sourceCode(context.packageName))
 
-    val additionalImports = GraphQLQueryGenerator.imports(context.packageName)
+    val customImports = {
+      def importer(c: List[String]): List[Importer] = {
```

Comment:
This can be a private method once moved to GraphQLQueryGenerator.
`c` is also a bit short as a function parameter. `imports` would be more readable.